scout/constants/variant_tags.py

```diff
@@ -49,6 +49,79 @@
 
 VARIANT_CALL = ("Pass", "Filtered", "Not Found", "Not Used")
 
+# Formatting for different filters in the vcf file
+VARIANT_FILTERS = {
+    "pass": {"label": "PASS", "description": "Passed filtering", "label_class": "success"},
+    "germline": {"label": "GERM", "description": "Germline variant", "label_class": "warning"},
+    "germline_risk": {"label": "GERM", "description": "Germline risk", "label_class": "warning"},
+    "fail_nvaf": {
+        "label": "N",
+        "description": "Too high VAF in normal sample",
+        "label_class": "danger",
+    },
+    "fail_pvalue": {"label": "P", "description": "Too low P-value", "label_class": "danger"},
+    "fail_homopolymer_indel": {
+        "label": "HP",
+        "Variant in homopolymer region": "Passed filtering",
+        "label_class": "danger",
+    },
+    "fail_longdel": {
+        "label": "LD",
+        "description": "Long DEL from vardict",
+        "label_class": "danger",
+    },
```

Comment:
We don't apply any length filter in VarDict on this one either. 

Generally, we trust deletions from VarDict as we still have sv calling enabled.